docs/final-touches.md

```diff
@@ -26,21 +26,51 @@ Please take a look at the privacy policy document; it is accessible for the user
 
+## Player action log
+
+:::caution
+This feature is accessible only for accounts with Superuser role set.
+:::
+
+Version 10.x comes up with player action log feature. This lets you see who was accessing the website, when and from what kind of browser a connection was made. It is possible to filter results by entering:
+
+- name,
+- Steam ID (SteamID64 format),
+- IP address,
+- User agent,
+- Action (connection with the website/game server(s)).
+
+All connections with the website and gameservers are logged.
+
+![player-action-log](/img/content/player-action-log.png)
+
 ## Adding game servers to the website
 
 Pickup games require game servers on which it can be set up. In order to do that, you have to define the same game server secret for the server (variable `GAME_SERVER_SECRET` in `.env`) and for game servers (variable `TF2PICKUPORG_SECRET` in `gameserver_x.env` or by defining `sm_tf2pickuporg_secret` value in game server's `server.cfg` configuration file). Server being behind a proxy [may need an internal address value defined](/docs/site-components-deployment#gameserver_1env).
 
-This mechanism is used since server version 8.x and the game server setup is being done automatically. After discovering newly setup game servers, they will show up on the server list:
+This mechanism is used since server version 8.x and the game server setup is being done automatically. After discovering newly setup game servers, they will show up on the server list, which can be accessed through Admin Panel -> Game servers:
 
-![game-server-status](/img/content/game-server-status.png)
+![game-servers-configuration](/img/content/game-servers-configuration.png)
 
-You can always use **Run diagnostics** button in order to perform server troubleshooting. The site will perform a tests to ensure the game server works correctly.
+If you have any static servers, they will show up on the list. You can click on any of them in order to see their address and to perform diagnostics. For that click **Run diagnostics** button in order to start. The site will perform tests to ensure the game server works correctly.
 
-![run-diagnostics](/img/content/run-diagnostics.png)
+![game-server-details](/img/content/game-server-details.png)
 
 There you can see the diagnostics status. In this case everything went smooth and things seem to be fine.
 
-![diagnostics-status](/img/content/diagnostics-status.png)
+![game-server-diagnostics](/img/content/game-server-diagnostics.png)
+
+:::info
+serveme.tf servers are being used in pickup games **only** if no static game servers are assigned to the site.
+:::
+
+In you have a valid serveme.tf API key set in your `.env` file, _serveme.tf integration settings_ becomes available:
+
+![configure-serveme-integration](/img/content/configure-serveme-integration.png)
+
+Currently the only setting you can define here is a preferred region for reserved servers, depending to which serveme.tf do you use (European, North American or Oceanic regions).
```

Comment:
Grammar:

> Currently the only setting you can define at this point is the preferred region of the reserved servers.

Also, the other part is kind of misleading, I'd rather leave it out.